virttest/qemu_vm.py

```diff
@@ -1696,15 +1697,16 @@ def __iothread_conflict_check(params):
                 raise ValueError(
                     "There's a conflict in the configuration! Once "
                     "'iothread_scheme' is set, 'image_iothread_vq_mapping' or"
-                    " 'iothreads' can NOT be set!")
+                    " 'iothreads' can NOT be set!"
+                )
```

Comment:
Those should be covered by https://github.com/avocado-framework/avocado-vt/pull/3785/commits/a304a406710d133b12a9e3301536362fac29b0bd